blockprod/src/rpc.rs

```diff
@@ -22,54 +22,59 @@ use common::{
 
 use serialization::{hex::HexDecode, hex::HexEncode};
 
-use crate::BlockProductionError;
+use crate::{detail::JobKey, BlockProductionError};
 use subsystem::subsystem::CallError;
 
 #[rpc::rpc(server, namespace = "blockprod")]
 trait BlockProductionRpc {
```

Comment:
I'm not sure if it's captured anywhere, but we may later want to restrict access to these RPC calls to prevent abuse from unauthed users.